packages/fuels-types/src/errors.rs

```diff
@@ -27,6 +27,8 @@ impl From<Utf8Error> for CodecError {
 
 #[derive(Error, Debug)]
 pub enum Error {
+    #[error("General error: {0}")]
+    GeneralError(String),
```

Comment:
I do not like the name `GeneralError` so much. I understand why you would give it as this error is constructed from any `str`.
The best solution would be if we could update the error type in `fuel-types`. Until then maybe we call it `FromStrError` ?

Reply:
I agree that `GeneralError` is not ideal. I wanted to give this some more thought, but it got lost in the process. On the other hand, I think going with `FromStrError` adds to the proliferation of one-off error types.